src/components/popovers/menu.stories.mdx

```diff
@@ -72,7 +72,7 @@ The menu will automatically position above or below the trigger element based on
 
 ## Trigger element
 
-The Menu trigger can be any JSX, but a `Button` with the `hasCaret` prop enabled is recommended.
+The Menu trigger can accept any focusable element, but a `Button` with the `hasCaret` prop enabled is recommended.
```

Comment:
could we also wrap the `trigger` example on line 90 with a `<button>` or link or focusable element?

Reply:
I really should have combed over this story 😓 sure I'll change this and I'll make sure the rest of this story makes sense.